docs-src/security/how-to-set-up-codec-server.md

```diff
@@ -12,7 +12,7 @@ To use a [Codec Server](/concepts/what-is-a-codec-server), first run it with you
 
 ### Run the server
 
-A Codec Server is an HTTP server that implements two endpoints:
+A Codec Server is an HTTP server that follows [this protocol](https://github.com/temporalio/samples-go/tree/main/codec-server#codec-server-protocol). It implements two endpoints:
```

Comment:
```suggestion
A Codec Server is an HTTP server that follows the Temporal [Codec Server Protocol](https://github.com/temporalio/samples-go/tree/main/codec-server#codec-server-protocol).
It implements two endpoints:
```
Changed link text to match target, per https://developers.google.com/style/link-text#structure-link-text.

Reply:
Dunno, maybe it should be changed to [here](https://twitter.com/dailmageejr/status/1613325603595505666?s=46&t=7CC01XA6eei5ycae7dS5PQ)